client/my-sites/plans-features-main/index.tsx

```diff
@@ -376,6 +377,8 @@ const PlansFeaturesMain = ( {
 		? 'plans-default-wpcom'
 		: intentFromProps || intentFromSiteMeta.intent || 'plans-default-wpcom';
 
+	const { isLoadingHostingTrialExperiment, isAssignedToHostingTrialExperiment } =
+		useFreeHostingTrialAssignment();
 	const eligibleForFreeHostingTrial = useSelector( isUserEligibleForFreeHostingTrial );
```

Comment:
Unfortunately there's too senses of the word "eligible".
You can be eligible to participate in the A/B test, but not eligible to get another free trial.

If you've already been through the free trial once then you are not eligible for _another_ one. But you _are_ eligible to be assigned to a cohort. In fact you must have already been assigned to a cohort, otherwise how'd you create your first trial site.

Maybe it'd be clearer if `eligibleForFreeHostingTrial` was called `eligibleToCreateFreeHostingTrial`